README.md

```diff
@@ -1,2 +1,8 @@
 # SYSC4806_Project
-The SYSC4806_Project For Group 5
+The SYSC4806_Project For Group 2
+
+# Mini-SurveyMonkey
+
+This project is a mini survey monkey application. It allows users to 
```

Comment:
Please add more within the readme, such as installation instructions and how to use the application

Reply:
That could be another pull request. We don't have enough yet for a readme.